pn.swagger.yaml

```diff
@@ -0,0 +1,253 @@
+        default:
+          description: Unexpected error.
+    delete:
+      tags:
+        - User
+      summary: remove a device token for push notifications
+      description: Remove a device/platform specific token for push notifications.
+      parameters:
+        - name: userToken
+          in: header
+          type: string
+        - name: body
+          in: body
+          schema:
+            $ref: '#/definitions/json_push_device'
+          description: push notification device data
+      responses:
+        '200':
+          schema:
+            type: object
+          headers: {}
+          description: Success
+        '204':
+          schema:
+            type: object
+          headers: {}
+          description: No change saved
+        default:
+          description: Unexpected error.
+
+  /push/android/{token}:
+    post:
+      tags:
+        - Push
+      summary: send push notification to Android (for dev/testing)
+      description: send a push notification to the Android device of the specified token
+      consumes:
+        - application/json
+      parameters:
+        - name: userToken
+          in: header
+          type: string
+        - name: token
+          in: path
+          required: true
+          type: string
+          description: token of the android device
+        - name: body
+          in: body
+          schema:
+            $ref: '#/definitions/json_unified_message'
+          description: push notification message
+      responses:
+        '200':
+          schema:
+            type: object
+          headers: {}
+          description: Success
+        '204':
+          schema:
+            type: object
+          headers: {}
+          description: No change saved
+        default:
+          description: Unexpected error.
+  /push/ios/{token}:
+    post:
+      tags:
+        - Push
+      summary: send push notification to iOS (for dev/testing)
+      description: send a push notification to the iOS device of the specified token
+      consumes:
+        - application/json
+      parameters:
+        - name: userToken
+          in: header
+          type: string
+        - name: token
+          in: path
+          required: true
+          type: string
+          description: token of the iOS device
+        - name: body
+          in: body
+          schema:
+            $ref: '#/definitions/json_unified_message'
+          description: push notification message
+      responses:
+        '200':
+          schema:
+            type: object
+          headers: {}
+          description: Success
+        '204':
+          schema:
+            type: object
+          headers: {}
+          description: No change saved
+        default:
+          description: Unexpected error.
+  /push:
+    post:
+      tags:
+        - Push
+      summary: send push notification to users
+      description: send a push notification to the registered devices of the given list of users
```

Comment:
This says "users" and "list of users" but there's only a single `userToken` and `userid`